<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WheelDiameterSet</class>
 <widget class="QWidget" name="WheelDiameterSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}</string>
  </property>
  <widget class="QLabel" name="lblOriginD2">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginC2">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginD1">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginB2">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginB1">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginA1">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginA2">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOriginC1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewB2">
   <property name="geometry">
    <rect>
     <x>696</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewA1">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewC2">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewD2">
   <property name="geometry">
    <rect>
     <x>518</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewB1">
   <property name="geometry">
    <rect>
     <x>251</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewA2">
   <property name="geometry">
    <rect>
     <x>785</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewC1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblNewD1">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnA1">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnB1">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnC1">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnD1">
   <property name="geometry">
    <rect>
     <x>434</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnD2">
   <property name="geometry">
    <rect>
     <x>523</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnC2">
   <property name="geometry">
    <rect>
     <x>612</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnB2">
   <property name="geometry">
    <rect>
     <x>701</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnA2">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>270</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>选中</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>原轮径值</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_38">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>轮径设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_39">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>160</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>220</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;</string>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblWarning">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>711</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 0px;
color: red;</string>
   </property>
   <property name="text">
    <string>设置失败，轮径值非法，新轮径值必须在[770mm，840mm]区间内！</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
